src/main/java/org/fcrepo/auth/webac/URIConstants.java

```diff
@@ -109,6 +109,12 @@
     public static final String WEBAC_ACCESS_CONTROL_VALUE = WEBAC_NAMESPACE_VALUE + "accessControl";
     public static final URI WEBAC_ACCESS_CONTROL = URI.create(WEBAC_ACCESS_CONTROL_VALUE);
 
+    /**
+     * FOAF Agent
+     */
+    public static final String FOAF_AGENT_VALUE = "http://xmlns.com/foaf/0.1/Agent";
```

Comment:
To make `foaf` more reusable, maybe you can use the pattern established above of defining the `foaf` namespace, then your `FOAF_AGENT_VALUE` can append to the namespace.